build/linux/installer/conf/telegraf-prom-side-car.conf

```diff
@@ -66,7 +66,15 @@
   ## Run telegraf in quiet mode (error log messages only).
   quiet = true
   ## Specify the log file name. The empty string means to log to stderr.
-  logfile = ""
+  logfile = "/var/opt/microsoft/docker-cimprov/log/telegraf_error.log"
+
+  ## Maximum number of rotated archives to keep, any older logs are deleted. If set to -1, no archives are removed.
+  logfile_rotation_max_archives = 3
```

Comment:
Do we need 3 log files as archives?

Reply:
> Do we need 3 log files as archives?

Yes, this is number of log files to keep and 3 should be fine with each 1MB. other places we have 5 - https://github.com/microsoft/Docker-Provider/blob/ci_prod/kubernetes/linux/main.sh#L940.